x11-packages/dosbox-x/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://dosbox-x.com/
+TERMUX_PKG_DESCRIPTION="DOSBox-X fork of the DOSBox project"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.83.22
+TERMUX_PKG_REVISION=1
```

Comment:
Remove this. This is only needed for changes made after package is published (for `dpkg` to consider it as update).